articles/extensions/authorization-extension/v2/implementation/configuration.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Configuring the Authorization Extension
+description: How to configure the Authorization Extension
+toc: true
+---
+
+# Authorization Extension: Configuration
+
+Before the extension can enforce your authorization logic, you'll need to configure how it will behave during the login transaction. Your configuration settings will be captured in a [rule](/rules) that's executed during runtime.
+
+## Configure the Extension
+
+Open up the Authorization Extension, and click **Configuration** on the drop-down menu in the top right of the **Authorization Dashboard**.
+
+![Click Configuration](/media/articles/extensions/authorization/click-configuration.png)
+
+This brings you to the **Rule Configuration** section of the **Configuration** page.
+
+![Configuration page](/media/articles/extensions/authorization/configuration.png)
+
+All of the changes you make in the sections under **Token Contents**, such as those related to groups, roles, and permissions, will be reflected in the rule you export at the completion of this step. 
+
+### Data Limitations
+
+Storing too much data in the token may cause performance issues or even prevent the issuance of the token. Be sure to store only what you need. If you need a large amount of user data readily available, consider using [persistence](#persistence) instead of adding the data to the token.
+
+## Configure Behavior for Groups- and Roles-Based Behavior and Permissions
+
+You can store authorization data like groups, roles, or permissions in the outgoing token issued by Auth0. Your application can then consume this information by inspecting the token and take appropriate actions based on the user's current authorization context.
+
+To add groups, roles, and/or permissions information to the outgoing token, simply enable the slider next to the option you want included.
+
+![Set token contents](/media/articles/extensions/authorization/user-info.png)
+
+### Merge and Use Groups, Roles, and Permissions from the IdP
+
+You might have users that receive groups, roles, or permissions from the identity provider (Id) you're using, such as Active Directory. If you want to merge these items (to preserve them) with the ones defined in the Authorization Extension, make sure you enable the appropriate **Passthrough** options. Simply enable the slider next to the appropriate merges you want enabled.
+
+![Enable passthroughs](/media/articles/extensions/authorization/passthrough.png)
+
+### Store Groups, Roles, and Permissions Information in the Users' Profiles
```

Comment:
Again maybe sth simpler like Persist Data in the User's Profile? No need to repeat `Groups, Roles, and Permissions`